src/main/java/com/worldwarofants/game/module/newgame/NewGameView.java

```diff
@@ -0,0 +1,21 @@
+package com.worldwarofants.game.module.newgame;
+
+import com.worldwarofants.game.arch.AbstractView;
+import com.worldwarofants.game.arch.console.ConsoleOutputHandler;
+
+public class NewGameView extends AbstractView<NewGameViewModel> {
+
+	public NewGameView(NewGameViewModel viewModel) {
+		super(viewModel);
+	}
+
+	public void renderNewGameScreen() {
+		ConsoleOutputHandler.lineBreak();
+		ConsoleOutputHandler.spell("-- GAME SETUP --");
+		ConsoleOutputHandler.lineBreak();
+		ConsoleOutputHandler.post("* Start Game - 'startGame'");
+		ConsoleOutputHandler.post("* Back - 'back'");
+		ConsoleOutputHandler.lineBreak();;
+		ConsoleOutputHandler.post("Please enter your command:");
+	}
```

Comment:
This whole method looks a bit freaky, right? It has the text `ConsoleOutputHandler` repeating nearly ten times, for such a simple output. There are pretty cool ways to simplify this. With the right implementation inside your output handler class, that code could look like this:
```
ConsoleOutputHandler.newOutput()
  .lineBreak()
  .spell("-- GAME SETUP --")
  .lineBreak()
  .post("* Start Game - 'startGame'")
  .post("* Back - 'back'")
  .lineBreak()
  .post("Please enter your command:")
  .print();
```

Where `newOutput()` initializes the process, and `print()` finalizes it, for example. This kind of method chaining can be done through the builder pattern. This one of the very popular and sweet looking approaches, which are called `fluent interfaces` https://en.wikipedia.org/wiki/Fluent_interface.

I would strongly encourage you to look through this, you might be able to make a very cool and useful class that everyone can use in their views for the project! It also provides plenty of room for optimizations, but we will get there with time.

Reply:
Awesome okay I'll look into that. Can we leave it as-is in this branch and I'll make a new branch just for ConsoleOutputHandler updates? It's gonna take me some time to wrap my head around this builder stuff.

Reply:
Of course! That is a very good approach, in fact better than doing those changes here, since they're a bit too big. Though you should leave a TODO in the current PR as a reminder to do this later.